### ISSUE TITLE
Extracting SevenNet descriptors

### ISSUE THREAD

**Issue by minhee2043** [MDIL-SNU/SevenNet#203]
Dear SevenNet team,

I am a member from Prof. Won Bo Lee group and I am currently using SevenNet as my main model (it had the best accuracy in interaction energies compared to other models like MACE and MatterSim!).

Since I will be fine-tuning the pretrained model, I am planning to sample meaningful configurations using dimension reduction method. So, I would like to ask if there is a simple way to extract the SevenNet descriptors. For example, MACE provides ASE based module of "get_descriptor", which users can easily obtain the descriptors that went through the message passing block. The details are in https://mace-docs.readthedocs.io/en/latest/guide/descriptors.html.

Thank you!

**Comment by YutackPark** [comment]
Thanks for choosing our model! I think that would be nice improvements of 7net.
Actually, I already have this repo: https://github.com/YutackPark/mace-mp-umap
Furthermore, you can see the usage of 7net descriptor from Fig. S15 of this paper: https://arxiv.org/abs/2501.05211 
However, the repo is not maintained and guide is missing. I'll let you know once the `get_descriptor` feature is officially introduced to 7net. Until then, this is the best I can share without much effort.

PS. the features I extracted includes only the last layer and its dimension is `N_atom * N_node_feats`.

**Comment by minhee2043** [comment]
Thank you Yutack for such a quick reply! I will have a look on your repository. :)
